drivers/storage/isilon/storage/isilon_storage.go

```diff
@@ -174,27 +174,20 @@ func (d *driver) getVolumeAttachments(ctx types.Context) (
 		return nil, err
 	}
 
-	// should be using this, but instanceID is coming back either blank or
-	// with metadata today
-	// iid := ctx.InstanceID()
-
-	ii, err := d.InstanceInspect(ctx, nil)
-	if err != nil {
-		return nil, err
-	}
-	iid := ii.InstanceID
-
-	ld, err := d.LocalDevices(ctx, nil)
-	if err != nil {
-		return nil, err
+	iid, iidOK := context.InstanceID(ctx)
+	if iidOK {
+		ld, err := d.LocalDevices(ctx, nil)
```

Comment:
Hi @Oskoss,

Thanks for the update, but you still need to update this line regarding the local devices function. Want to set up a Google Hangout? I think it should be fairly straight-forward. Use the `context.LocalDevices` function instead.